.travis.yml

```diff
@@ -24,26 +18,47 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported PHP and Symfony version
+        - php: 7.1
+          env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="composer test-ci" SYMFONY_DEPRECATIONS_HELPER="weak" SYMFONY_PHPUNIT_VERSION="5.7"
+
+          # Test the latest stable release
         - php: 5.5
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
-        - php: 7.2
-          env: DEPENDENCIES=dev
-        - php: hhvm
-          dist: trusty
-          # Test against LTS versions
+          env: SYMFONY_PHPUNIT_VERSION="5.7"
+        - php: 5.6
+          env: SYMFONY_PHPUNIT_VERSION="5.7"
         - php: 7.0
-          env: SYMFONY_VERSION=2.8.*
+        - php: 7.1
+        - php: 7.2
+
+          # Test LTS versions
+        - php: 5.5
+          env: DEPENDENCIES="symfony/lts:^2"
+        - php: 5.5
+          env: DEPENDENCIES="symfony/lts:^3"
```

Comment:
This allows some dependencies to be installed in their 2.x version. Is that wanted?

Reply:
Yes, That is fine.